salt/modules/zypper.py

```diff
@@ -21,6 +21,10 @@
 import os
 import time
 import datetime
+try:
+    from salt.utils.versions import LooseVersion
+except ImportError:
+    LooseVersion = None
```

Comment:
@terminalmage this is shorter version of:
```python
try:
   import something
   HAVE_SOMETHING = True
except ImportError:
   HAVE_SOMETHING = FALSE
...
if HAVE_SOMETHING: ...
```

But yeah, you're right: I just fixed lint and noticed that this just there forever.